src/affine.jl

```diff
@@ -202,5 +202,9 @@ function Base.:(==)(t1::LinearMap, t2::AffineMap)
         0 == vecnorm(t2.v)
 end
 
+function center(trans::LinearMap, origin::AbstractVector)
```

Comment:
I think we could expand `LinearMap` to arbitrary transformations, no? 

For `LinearMap` this will nicely compose to an `AffineMap` automatically anyway.